Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/cytoscapeobjects/CytoscapeLinkDataObject.java

```diff
@@ -0,0 +1,62 @@
+package edu.asu.spring.quadriga.web.publicwebsite.cytoscapeobjects;
```

Comment:
bad package name. 'publicwebsite' is already not very good, but adding 'cytoscapeobjects' to it makes it even worse. Find a short name that's to the point.